ledger/internal/cow.go

```diff
@@ -132,6 +134,11 @@ func (cb *roundCowState) deltas() ledgercore.StateDelta {
 			}
 		}
 	}
+	return cb.mods
+}
+
+func (cb *roundCowState) deltas() ledgercore.StateDelta {
```

Comment:
this should be done another way around

Reply:
What exactly do you mean by that?

Reply:
usually exported func calls unexported

Reply:
Ok got it. I think now we can make both unexported